view/frontend/web/js/view/payment/method-renderer/cardknox-method.js

```diff
@@ -171,19 +214,22 @@ define(
                             //perform your own validation here...
                             if (document.getElementsByName("xCardNum")[0].value === '') {
                                 self.showError("Card Number Required");
+                                console.log("Card");
```

Comment:
Is this log still needed?
If yes, can you make it more descriptive?
Same for the log below